app/api/dao/admin.py

```diff
@@ -65,7 +65,10 @@ def revoke_admin_user(user_id: int, data: Dict[str, str]):
         """
         admin_user_id = data["user_id"]
 
-        if user_id == admin_user_id:
+        users_list = UserModel.query.filter(UserModel.id != user_id).all()
+        list_of_users = [user.json() for user in users_list if user.is_admin]
```

Comment:
Can you do this in the filter option in the line above? @lmcicat 
Generating this additional list seems unnecessary, you could filter by `UserModel.is_admin == true`, and see if the list returns at least 2 results.

Reply:
fixed :) also switched the query to be `.count()` rather than `.all()` since we only care about the number of users returned and not the users themselves.

Reply:
thats great catch :D @lmcicat